apps/projects/overwrites.py

```diff
@@ -0,0 +1,12 @@
+from django.utils.translation import ugettext_lazy as _
+
+from adhocracy4.projects.enums import Access
+
+
+def overwrite_access_enum_label():
+    Access.__labels__ = {
+        Access.PRIVATE: _('Only invited users can participate (private).'),
+        Access.PUBLIC: _('All users can participate (public).'),
+        Access.SEMIPUBLIC: _('All users can view the content but only invited '
+                             'users can participate (semipublic).')
+    }
```

Comment:
The labels in the story are a little different: https://taiga.liqd.net/project/liqd-liquid-software/us/3433?milestone=239
Sorry :see_no_evil:

Reply:
Adopted :)

Reply:
I am sooo sorry, but no :) Its the wording below the acceptance criteria (wording in basic settings:)..

Reply:
Oh, didn't see that, thanks!

concerning review: IMO you shouldn't be sorry for that - by properly reviewing the PR you do me and all of us a big favour, saving us time and effort afterwards. Right now it's easy to fix up, costing me way less effort than having somebody to open an issue etc. So please, rather nitpick often :)

Reply:
:+1: